src/Microsoft.Extensions.ProjectModel.Sources/IProjectContext.cs

```diff
@@ -24,5 +25,7 @@ public interface IProjectContext
         IEnumerable<string> CompilationItems { get; }
         IEnumerable<string> EmbededItems { get; }
         string FindProperty(string propertyName);
+        IEnumerable<DependencyDescription> Dependencies { get;}
+        IEnumerable<ResolvedReference> ResolvedReferences { get; }
```

Comment:
Can you clarify what's in this list? I'm guessing it's the sum of {ProjectReference, PackageReference, Reference (GAC or local) }